src/entry.js

```diff
@@ -1,22 +1,17 @@
-import calculator from "./calculator";
+import Desmos from "./calculator";
 
-const script = document.createElement("script");
```

Comment:
I removed the script generating thing. I wasn't totally sure if this was desired for some reason, or was just part of an attempt to get the system to work.